docs/source/federated-types/federated-directives.mdx

```diff
@@ -27,10 +27,10 @@ If an imported directive's default name matches one of your own custom directive
 ```graphql
 extend schema
   @link(url: "https://specs.apollo.dev/federation/v2.3",
-        import: [{ name: "@key", as: "@primaryKey"}, "@shareable"])
+        import: [{ name: "@key", as: "@uniqueKey"}, "@shareable"])
```

Comment:
I originally wasn't going to change this renaming directives example since users can rename a directive however they wish. I decided to in the end for consistency purposes. Curious if folks feel strongly otherwise though.

Reply:
I see no reason not to